### DISCUSSION TITLE
ScriptReference/Playables.PlayableDirector-time

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#268] [Notes]
# ScriptReference/Playables.PlayableDirector-time

https://docs.unity3d.com/ScriptReference/Playables.PlayableDirector-time.html

**Comment by radiatoryang** [comment]
note: this value goes back to 0 after it finishes playing a Playable, even if its WrapMode (extrapolationMode) is set to "None"... to detect when a PlayableDirector is done playing, I do a little hack:   

```csharp
playableDirector.Play();  
playableDirector.time = 0.001;  
...  
if ( playableDirector.time <= 0 ) // director is finished 
```
